src/Host/Client/Impl/Debugging/IDebuggerModeTracker.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.R.Components.InteractiveWorkflow {
+namespace Microsoft.R.Host.Client.Debugging {
```

Comment:
I'd rather define `CanInterrupt` in `Components` assembly and make it extension to `IDebuggerModeTracker`.

Reply:
But it is session we are interrupting. The fact that it checks the debugger state is implementation detail.